index.html

```diff
@@ -2309,6 +2309,112 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double latency
+          </dt>
+          <dd>
+            <p>
+              The estimation in seconds of audio output latency, i.e. the delay
+              between the moment the implementation sends a buffer to the
+              system and the audible sound from the audio output (speaker or
+              connector).
```

Comment:
I think this language is a little vague though the intent is clear. My suggestion would be:

The estimation in seconds of audio output latency, i.e. the interval between the time
the UA requests the host system to play a buffer and the time at which the first sample in the buffer
is actually processed by the audio output device. For audible devices such as speakers or headphones
that produce an acoustic signal, this latter time refers to the time when a sample's sound is produced.